scripts/sbiParser.js

```diff
@@ -33,7 +33,7 @@ export class sbiParser {
     static #damageTypesRegex = /\bbludgeoning\b|\bpiercing\b|\bslashing\b|\bacid\b|\bcold\b|\bfire\b|\blightning\b|\bnecrotic\b|\bpoison\b|\bpsychic\b|\bradiant\b|\bthunder\b/ig;
     static #conditionTypesRegex = /\bblinded\b|\bcharmed\b|\bdeafened\b|\bdiseased\b|\bexhaustion\b|\bfrightened\b|\bgrappled\b|\bincapacitated\b|\binvisible\b|\bparalyzed\b|\bpetrified\b|\bpoisoned\b|\bprone\b|\brestrained\b|\bstunned\b|\bunconscious\b/ig;
     static #sensesRegex = /(?<name>\bdarkvision\b|\bblindsight\b|\btremorsense\b|\btruesight\b) (?<modifier>\d+)/i;
-    static #challengeRegex = /^(challenge|cr|challenge rating)\s?(?<cr>(½|[\d\/]+))\s?(\((?<xp>[\d,]+)\s?xp\))?/i;
+    static #challengeRegex = /^(challenge|cr|challenge rating)\s?(?<cr>(½|[\d\/]+))(\s(?<role>\w+))?\s?(\((?<xp>[\d,]+)\s?xp\))?/i;
```

Comment:
I'll have to look into this. I don't like the idea of the user needing to modify the statblock in order for it to parse correctly. There's no obvious way they'd know to do that. If we want to support Flee, Mortals! statblocks then we should do it in a way that "just works".

Reply:
Yea I definitely agree that would be ideal, unfortunately I'm not really sure where to get started with that one, since the CR is always at the bottom of the statblock due to their formatting, so it's usually getting swallowed into the last feature listed.

Reply:
Also I guess I should note that this is what users are currently doing, unless they are just ignoring those lines and manually updating them.
Also I was able to get Villain Actions parsed into separate items, but that change has more substantial implications (changes to #actionTitleRegex) so I will file that in a separate PR to review